docs/imagej_python.rst

```diff
@@ -88,7 +88,11 @@ One of the advantage of this approach is that we can load only the 2D-planes we
 The script used in this document contains an ImageJ1 macro that needs graphical user interface (GUI) elements, and thus it requires using ImageJ in GUI mode. In this GUI mode, the resulting windows content is handled.
 If you are running the example in the Docker container, 
 you will also need to start UI environment if it is not already up.
-If you have used the option 2 above, select ``desktop`` to start the UI environment. If you do not use any ImageJ1 features e.g. macro, you do **not** need the UI environment and do **not** need to set the `headless` parameter (default is `True`).
+If you have used the option 2 above, select ``desktop`` to start the UI environment.
+
+    \ |image0|
+
+If you do not use any ImageJ1 features e.g. macro, you do **not** need the UI environment and do **not** need to set the `headless` parameter (default is `True`).
```

Comment:
```suggestion

```

Reply:
This sentence - why do we need that ? It still creates confusion.

Reply:
If someone is not interested to run the macro then this is not needed.
I could write a second script and point to it.